pkg/proxy/integrations/mongoparser/mongoparser.go

```diff
@@ -596,7 +601,9 @@ func isHeartBeat(opReq Operation, requestHeader models.MongoHeader, mongoRequest
 	case wiremessage.OpMsg:
 		_, ok := mongoRequest.(*models.MongoOpMessage)
 		if ok {
-			return (opReq.IsIsAdminDB() && (strings.Contains(opReq.String(), "hello") || (isScramAuthRequest(mongoRequest.(*models.MongoOpMessage).Sections, logger)))) || opReq.IsIsMaster()
+			return (opReq.IsIsAdminDB() && strings.Contains(opReq.String(), "hello")) ||
```

Comment:
Please correct the indentation

Reply:
Indentation is correct. I have done this because line was getting bigger. And it is a good practice.☺️

Reply:
Got it.